composer.json

```diff
@@ -1,5 +1,20 @@
 {
     "require": {
-        "google/apiclient": "^2.5"
+        "php": ">=7.2.5",
+        "google/apiclient": "^2.5",
+        "google/apiclient-services": "v0.138"
+    },
+    "replace": {
+        "monolog/monolog": "*"
+    },
+    "scripts": {
+        "post-update-cmd": "Google\\Task\\Composer::cleanup"
+    },
+    "extra": {
+        "google/apiclient-services": [
+            "Analytics",
+            "AnalyticsReporting",
+            "Oauth2"
+        ]
```

Comment:
This removes all google services not listed

Reply:
Should the cleanup command also run on install?

Reply:
That's how Google suggest to implement it, and it should also run on install.

Reply:
When I deleted vendor and ran `composer install` I saw all the services when I ran `git status`. Can you try and tell me what you see?

Reply:
hm... just checked that again and seems for a clean install it wasn't executed... Added it on install now as well.